src/System.Collections.NonGeneric/src/System/Collections/SortedList.cs

```diff
@@ -472,8 +470,7 @@ public virtual IList GetKeyList()
         // 
         public virtual IList GetValueList()
         {
-            if (_valueList == null) _valueList = new ValueList(this);
-            return _valueList;
+            return _valueList ?? (_valueList = new ValueList(this));
```

Comment:
Please revert this change but do feel free to put this if block on its own line to make it match the coding guidelines.